tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/test_formula_5_4.py

```diff
@@ -0,0 +1,97 @@
+"""Testing formula 5.4 of NEN-EN 1992-1-1+C2:2011."""
+
+import pytest
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011.chapter_5_structural_analysis.formula_5_4 import Form5Dot4TTransverseForceEffectBracingSystem
+from blueprints.validations import NegativeValueError
+
+
+class TestForm5Dot4TTransverseForceEffectBracingSystem:
+    """Validation for formula 5.4 from NEN-EN 1992-1-1+C2:2011."""
+
+    def test_evaluation(self) -> None:
+        """Test the evaluation of the result."""
+        # Example values
+        theta_i = 0.003  # -
+        n_a = 5  # kN
+        n_b = 10  # kN
+
+        # Object to test
+        form_5_4 = Form5Dot4TTransverseForceEffectBracingSystem(theta_i=theta_i, n_a=n_a, n_b=n_b)
+
+        # Expected result, manually calculated
+        manually_calculated_result = 0.015  # kN
+
+        assert form_5_4 == pytest.approx(expected=manually_calculated_result, rel=1e-4)
+
+    def test_raise_error_when_negative_theta_i_is_given(self) -> None:
+        """Test a negative value for theta_i."""
+        # Example values
+        theta_i = -0.003
+        n_a = 5
+        n_b = 10
+
+        with pytest.raises(NegativeValueError):
+            Form5Dot4TTransverseForceEffectBracingSystem(theta_i=theta_i, n_a=n_a, n_b=n_b)
+
+    def test_raise_error_when_negative_n_a_is_given(self) -> None:
+        """Test a negative value for n_a."""
+        # Example values
+        theta_i = 0.003
+        n_a = -5
+        n_b = 10
+
+        with pytest.raises(NegativeValueError):
+            Form5Dot4TTransverseForceEffectBracingSystem(theta_i=theta_i, n_a=n_a, n_b=n_b)
+
+    def test_raise_error_when_negative_n_b_is_given(self) -> None:
+        """Test a negative value for n_b."""
+        # Example values
+        theta_i = 0.003
+        n_a = 5
+        n_b = -10
+
+        with pytest.raises(NegativeValueError):
+            Form5Dot4TTransverseForceEffectBracingSystem(theta_i=theta_i, n_a=n_a, n_b=n_b)
```

Comment:
You can consider using parametrize (https://docs.pytest.org/en/7.1.x/how-to/parametrize.html). Here's how it would look like:

```suggestion
    @pytest.mark.parametrize(
        ("theta_i", "n_a", "n_b"),
        [
            (-0.003, 5, 10),
            (0.003, -5, 10),
            (0.003, 5, -10),
        ],
    )
    def test_raise_error_when_negative_theta_i_is_given(
        self, theta_i: float, n_a: float, n_b: float
    ) -> None:
        """Test negative values for theta_i, n_a and n_b."""
        with pytest.raises(NegativeValueError):
            Form5Dot4TTransverseForceEffectBracingSystem(
                theta_i=theta_i, n_a=n_a, n_b=n_b
            )
```